JSQMessagesViewController/Views/MEGAInputToolbar.m

```diff
@@ -758,4 +745,22 @@ - (void)longPress:(UILongPressGestureRecognizer *)longPressGestureRecognizer {
     }
 }
 
+#pragma mark - Private methods.
+
+- (BOOL)handleAVAudioSessionError:(NSError *)error {
+    AVAudioSessionErrorCode errorCode = error.code;
+    NSString *errorMessage;
+    switch (errorCode) {
+        case AVAudioSessionErrorCodeInsufficientPriority:
+            errorMessage = AMLocalizedString(@"It is not possible to record voice messages while there is a call in progress", @"Message shown when there is an ongoing call and the user tries to record a voice message");
+            break;
+            
+        default:
+            errorMessage = [NSString stringWithFormat:@"Error recording voice message: %td", errorCode];
```

Comment:
I completely understand that this needs to be localized. I have moved this piece of code from `396` to `408`. If this issue is not urgent, I can update the PR on Monday.

Reply:
Displaying the localized description instead of text with error code.